mantidproject/mslice
# Background Subtraction

**Not to be merged until after #220**

When on the unprojected workspaces tab, the `subtract` button can be used to background subtraction. It produces a dialog that asks for a workspace to subtract and the self-shielding factor.

**To test:**
 - Load one or more unprojected workspaces
- Go to workspace manager
- Select one or more workspaces
- Click subtract
- Input a workspace to subtract and SSF into the dialog and press OK
- New workspace should appear with '_subtracted' appended to the name.

Fixes #221
